src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/Basic_Shapes.xaml.cs

```diff
@@ -0,0 +1,651 @@
+
+					return BuildHoriVertTestGridForScreenshot(shape, alterators);
+				}
+				catch (Exception error)
+				{
+					return new TextBlock {Text = $"Failed to render {parsedId}: {error.Message}", Foreground = new SolidColorBrush(Colors.Red)};
+				}
+			}
+		}
+
+		private static FrameworkElement BuildHoriVertTestGridForScreenshot(Factory shape, Alterator[] alterators)
+		{
+			var grid = BuildHoriVertTestGrid(
+				() => alterators.Aggregate(shape.Create(), (s, a) =>
+				{
+					a.Alter(s);
+					return s;
+				}),
+				null,
+				150);
+
+			grid.Background = new SolidColorBrush(Colors.White); // Much easier for screenshot comparison :)
+			grid.VerticalAlignment = VerticalAlignment.Top;
+			grid.HorizontalAlignment = HorizontalAlignment.Left;
+
+			// We set clip and add wrapping container so the rendering engine won't generate screenshots with a transparent padding.
+			grid.Clip = new RectangleGeometry {Rect = new Rect(0, 0, grid.Width, grid.Height)};
+			var containerGrid = new Grid
+			{
+				Width = grid.Width,
+				Height = grid.Height,
+				Children = {grid}
+			};
+
+			return containerGrid;
+		}
+
+		private static Grid BuildHoriVertTestGrid(Func<FrameworkElement> template, string title, int itemSize = 150)
+		{
+			var isLabelEnabled = title != null;
+			var horizontalAlignments = new[] {HorizontalAlignment.Left, HorizontalAlignment.Center, HorizontalAlignment.Right, HorizontalAlignment.Stretch};
+			var verticalAlignments = new[] {VerticalAlignment.Top, VerticalAlignment.Center, VerticalAlignment.Bottom, VerticalAlignment.Stretch};
+			var grid = new Grid();
+			var itemDimensions = new Size(itemSize + 2, itemSize + 20 + 2);
+			(int x, int y) labels;
+			if (isLabelEnabled)
+			{
+				grid.RowDefinitions.Add(new RowDefinition {Height = GridLength.Auto});
+				grid.RowDefinitions.Add(new RowDefinition {Height = GridLength.Auto});
+				grid.ColumnDefinitions.Add(new ColumnDefinition {Width = new GridLength(75)});
+
+				labels = (1, 2);
+
+				grid.Children.Add(GetLabel(title).GridColumnSpan(horizontalAlignments.Length + 1));
+				grid.Children.AddRange(horizontalAlignments.Select((h, x) => GetLabel(h).GridRow(1).GridColumn(x + labels.x)));
+				grid.Children.AddRange(verticalAlignments.Select((v, y) => GetLabel(v).GridRow(y + labels.y).GridColumn(0)));
+			}
+			else
+			{
+				labels = (0, 0);
+
+				// We hard-code width and height for screenshots:
+				// as content might overflow a bit, the rendering engine will append transparent padding in screenshots.
+				grid.Width = horizontalAlignments.Length * itemDimensions.Width;
+				grid.Height = verticalAlignments.Length * itemDimensions.Height;
+			}
+
+			grid.ColumnDefinitions.AddRange(horizontalAlignments.Select(_ => new ColumnDefinition {Width = new GridLength(itemDimensions.Width)}));
+			grid.RowDefinitions.AddRange(verticalAlignments.Select(_ => new RowDefinition {Height = new GridLength(itemDimensions.Height)}));
+
+			for (var x = 0; x < horizontalAlignments.Length; x++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5619334)